uitools/common/src/AuthenticatorController.cpp

```diff
@@ -143,7 +143,8 @@ void AuthenticatorController::handleArcGISAuthenticationChallenge(ArcGISAuthenti
           return;
         }
 
-        m_currentArcGISChallenge->continueAndFailWithError(e.error());
+        emit previousFailureCountChanged();
+        m_currentArcGISChallenge->continueWithError(e.error());
         m_currentArcGISChallenge.reset();
```

Comment:
Does this code path need the same fix of `deleteLater`? I assume this will crash in the OAuth workflow without your fix.